src/main/fc/cli.c

```diff
@@ -374,8 +374,8 @@ typedef enum {
 #ifdef BLACKBOX
     TABLE_BLACKBOX_DEVICE,
 #endif
-    TABLE_CURRENT_SENSOR,
-    TABLE_BATTERY_SENSOR,
+    TABLE_CURRENT_METER,
```

Comment:
ok, so you can have many types of sensors, and metering adds additional requirements, mainly filtering or precision concerns.  I made a note of this at the top of battery.c